Formula/drafter.rb

```diff
@@ -0,0 +1,26 @@
+class Drafter < Formula
+  desc "Snow Crash parser harness"
```

Comment:
what does that mean

Reply:
It's the description of the source repo.

Reply:
I'm still not sure what it means. What is this?

Reply:
https://apiblueprint.org/tools.html says "Native C/C++ API Blueprint Parser" which is a little more intelligible

Reply:
Snow Crash is the reference parser (library) for API Blueprint, and this tool is a wrapper around that that makes it into a CLI tool and exposes header files and whatnot.